ui/components/app/qr-hardware-popover/qr-hardware-popover.js

```diff
@@ -45,15 +45,15 @@ const QRHardwarePopover = () => {
   const signRequestCancel = useCallback(() => {
     let action = cancelTx;
     switch (_txData.type) {
-      case MESSAGE_TYPE.PERSONAL_SIGN: {
+      case ApprovalType.PersonalSign: {
```

Comment:
Not strictly about approvals so these can stay as `MESSAGE_TYPE` as it's just categorising the transaction.